libvirt/pool.go

```diff
@@ -0,0 +1,113 @@
+package libvirt
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	libvirt "github.com/libvirt/libvirt-go"
+)
+
+const (
+	poolExistsID    = "EXISTS"
+	poolNotExistsID = "NOT-EXISTS"
+)
+
+// poolExists returns "EXISTS" or "NOT-EXISTS" depending on the current pool existence
```

Comment:
question here: are all this methods specific for the `libvirt` type? are we sure for another type they works? or required other flags.

Reply:
I'm not sure I understand the question. This method is for libvirt pool, yes.

Reply:
ok thx